backend/associations/tests/event/test_event.py

```diff
@@ -126,12 +126,28 @@ def test_if_event_admin_then_cannot_update_event_with_inconsistent_dates(self):
         event_after = Event.objects.get(pk=0)
         self.assertEqual(event_before, event_after)
 
-    def test_if_event_admin_then_can_update_participants(self):
+    def test_if_event_admin_then_cannot_update_participants(self):
+        participants_before = set(
+            [
+                x["participants"]
+                for x in Event.objects.filter(pk=0).values("participants")
+            ]
+        )
+
         self.login("17event_biero")
         data = {"pk": 0, "participants": ["17event_biero", "17member_biero"]}
         res = self.update(data["pk"], "biero", data=data)
         self.assertStatusCode(res, 200)
```

Comment:
The thing is, you are still allowed to edit anything BUT the participants.
And with DRF, providing extra fields, even if they are not in the serializer, does not yield a 403.
What the test does is asking to update the event `0` with a field which is not read (because not in the serializer); so it's as if you were asking to PATCH with an empty payload.
It's OK for me to have a 200.

Reply:
In that case, add a comment to specify  it